src/test/java/org/snakeyaml/engine/usecases/inherited/InheritedImportTest.java

```diff
@@ -29,29 +21,39 @@
 import org.snakeyaml.engine.v2.scanner.StreamReader;
 import org.snakeyaml.engine.v2.utils.TestUtils;
 
+import java.io.File;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 
 public abstract class InheritedImportTest {
 
-  public static final String PATH = "inherited_yaml_1_1";
+  public static final String PATH = "/inherited_yaml_1_1";
 
 
   protected String getResource(String theName) {
-    return TestUtils.getResource(PATH + File.separator + theName);
+    //noinspection InjectedReferences
+    return TestUtils.getResource(PATH + "/" + theName);
   }
```

Comment:
IntelliJ didn't understand the path reference when it's based on a variable, so I disabled the injection for this method